pkg/lclient/containers.go

```diff
@@ -74,6 +74,15 @@ func (dc *Client) RemoveContainer(containerID string) error {
 	return nil
 }
 
+// RemoveVolume removes a volume with the specified volume ID
+func (dc *Client) RemoveVolume(volumeID string) error {
```

Comment:
I see no unit tests written for this function :(

Reply:
Good catch, unit tests have been added to `containers_test.go`.